ide/atom/mac.md

```diff
@@ -0,0 +1,29 @@
+# [Atom](../atom)
+
+## Duplicate selection: <kbd>⌘</kbd><kbd>d</kbd>
```

Comment:
👍 I actually never use this shortcut, but I need to start.

It's <kbd>ctrl</kbd><kbd>d</kbd> in windows